pxr/usd/usdLux/lightAPI.h

```diff
@@ -46,6 +46,41 @@ class SdfAssetPath;
 /// type (e.g. RectLight or DistantLight) or is applied directly to a Gprim 
 /// that should be treated as a light.
 /// 
+/// <b>Quantities and Units</b>
+/// 
+/// Most renderers consuming OpenUSD today are RGB renderers, rather than
+/// spectral. Units in RGB renderers are tricky to define as each of the red,
+/// green and blue channels transported by the renderer represents the
+/// convolution of a spectral exposure distribution, e.g. CIE Illuminant D65,
+/// with a sensor response function, e.g. CIE 1931 𝓍̅. Thus the main quantity
+/// in an RGB renderer is neither radiance nor luminance, but "integrated
+/// radiance" or "tristimulus weight".
+/// 
+/// The emission of a default light with `intensity` 1 and `color` [1, 1, 1] is
+/// an Illuminant D spectral distribution with chromaticity matching the
+/// rendering color space white point, normalized such that a ray normally
+/// incident upon the sensor with EV0 exposure settings will generate a pixel
+/// value of [1, 1, 1] in the rendering color space.
+/// 
+/// Given the above definition, that means that the luminance of said default
+/// light will be 1 *nit (cd∕m²)* and its emission spectral radiance
+/// distribution is easily computed by appropriate normalization.
```

Comment:
Watts and Joules >>> lumens, nits and candelas

Reply:
In this case, not really. If you want to talk about the radiance here, what wavelength should we talk about it at? How do we tie that to an RGB pixel colour? Or should we talk about the integrated radiance over some range of wavelengths?

Reply:
Yes, here, and with all the radiometric comments, I was thinking about integrated radiance over some range. I will follow up.

Reply:
I think the answer to my question is "you are doubilng down on the boneheaded misinterpretation in your previous question. please recall, where we specify a D series illuminant, and the sensor response function from which radiometric values may be recovered"

Reply:
:D